### ISSUE TITLE
Compare Size of MBTiles

### ISSUE THREAD

**Issue by lukasmartinelli** [osm2vectortiles/osm2vectortiles#77]
- We should try hard to make the MBTiles file smaller.
- We need to compare our MBTiles with the v6 Mapbox Streets.

**Comment by klokan** [comment]
My previous [comment](https://github.com/osm2vectortiles/osm2vectortiles/issues/11#issuecomment-158440594):
>  We should put an effort into improvement of the tile size.
It would be also interesting to compare the above mentioned big tiles with V6 - to identify why are our so large. It can be easily done with:
http://klokantech.github.io/ol3-sandbox/vector/tile-inspector.html#http://a.tiles.mapbox.com/v4/mapbox.mapbox-streets-v6.json?access_token=pk.eyJ1Ijoib3NtMnZlY3RvcnRpbGVzIiwiYSI6ImNpZnlkb2d0dTAyaDd1M2x6ZGg5cGp4aXQifQ.74RX5npQKmCE1jxsbXYNzQ|0|0|0

**Comment by manuelroth** [comment]
We did some analysis of our vector tiles, therefore I created two gists one which compares the size of mapbox-streets-v6 with open-streets and another which compares the features.

size compare: https://gist.github.com/manuelroth/ae138de3155519512bd1/revisions
feature compare: https://gist.github.com/manuelroth/28fadbed722e3f1ce633/revisions

Results:

| Zoom level        |   Diffs (os: open-streets, v6: streets-v6) |
| ------------- |:-------------|
| 5    | too many roads (open-streets: 41066 features / streets-v6: 6 features), don' show type trunk   |
| 6    | too many roads (os: 23221 / v6: 12), layer poi_label, road_label missing    |
| 7    | too many roads (os: 35185 / v6: 11), layer poi_label, road_label missing    |
| 8    | too many road_labels (os: 3575 / v6: 17), layer poi_label missing    |
| 9    | too many road_labels (os: 1611 / v6: 19)    |
| 10    | layer poi_label missing    |
| 11    | too many road_labels (os: 1936 / v6: 47)    |
| 12    | too many road_labels (os: 5191 / v6: 26)    |
| 13    | too many roads(os: 2986 / v6: 21), too many road_labels (os: 2027 / v6: 54)    |
| 14    | too many buildings (os: 3227 / v6: 1624), too many roads (os: 2343 / v6: 995), too many road_labels (os: 1637/ v6: 45)    |

It seams that our main problems are the layers roads and road_labels, we show too many of them. Of course we can see a lot more differences in the gist, but I think these are the main reason for the bigger size of our vector tiles.

**Comment by lukasmartinelli** [comment]
Thanks for the analysis!! We are several factors larger in for most tile indizes.

Roads should be possible to improve in a next release. Perhaps the roads are not filtered simply by class like we assumed but also filtered by other criterias.

Funny that this slipped through - as I remembered we went through each zoom level for the roads and tried to bring it as close to Mapbox as possible.

At **zoom level 14 it is difficult to throw away any** buildings, roads and road labels because we do not have a `z15` that can contain them like Mapbox does.
